modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -14251,6 +14516,78 @@ private void _testGetObjectEntryWithObjectActions(
 		unsafeTriConsumer.accept(actionJSONObject, jsonObject, objectAction);
 	}
 
+	private void _testPatchPutScopeScopeKeyByExternalReferenceCode(
+			Http.Method httpMethod)
+		throws Exception {
+
+		_siteScopedObjectEntry1 = ObjectEntryTestUtil.addObjectEntry(
+			_siteScopedObjectDefinition1, _OBJECT_FIELD_NAME_1,
+			_OBJECT_FIELD_VALUE_1);
+		Group group = _groupLocalService.fetchGroup(
+			TestPropsValues.getGroupId());
```

Comment:
If we want to follow the last suggestion, we can get the `groupId` by the object entry.